core/chains/evm/forwarders/orm.go

```diff
@@ -1,105 +1,108 @@
 package forwarders
 
 import (
+	"context"
 	"database/sql"
 
+	"github.com/smartcontractkit/chainlink-common/pkg/sqlutil"
+
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/jmoiron/sqlx"
 	"github.com/pkg/errors"
 
-	"github.com/smartcontractkit/chainlink-common/pkg/logger"
 	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/utils/big"
-	"github.com/smartcontractkit/chainlink/v2/core/services/pg"
 )
 
 //go:generate mockery --quiet --name ORM --output ./mocks/ --case=underscore
 
 type ORM interface {
-	CreateForwarder(addr common.Address, evmChainId big.Big) (fwd Forwarder, err error)
-	FindForwarders(offset, limit int) ([]Forwarder, int, error)
-	FindForwardersByChain(evmChainId big.Big) ([]Forwarder, error)
-	DeleteForwarder(id int64, cleanup func(tx pg.Queryer, evmChainId int64, addr common.Address) error) error
-	FindForwardersInListByChain(evmChainId big.Big, addrs []common.Address) ([]Forwarder, error)
+	CreateForwarder(ctx context.Context, addr common.Address, evmChainId big.Big) (fwd Forwarder, err error)
+	FindForwarders(ctx context.Context, offset, limit int) ([]Forwarder, int, error)
+	FindForwardersByChain(ctx context.Context, evmChainId big.Big) ([]Forwarder, error)
+	DeleteForwarder(ctx context.Context, id int64, cleanup func(tx sqlutil.Queryer, evmChainId int64, addr common.Address) error) error
+	FindForwardersInListByChain(ctx context.Context, evmChainId big.Big, addrs []common.Address) ([]Forwarder, error)
 }
 
-type orm struct {
-	q pg.Q
+type DbORM struct {
```

Comment:
Do we have to export this? I guess it should be `DBORM` then? But the name conflict only arises because we have a non-idiomatic habit of declaring interfaces where we don't need to, and that is done with an expectation that we will return the interface from the constructor instead of the concrete type, which can remain unexported. By exporting this, we are merging two conflicting styles.